diff --git a/configure.ac b/configure.ac
index b09d51a..a8979a3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -47,7 +47,7 @@ TORRENT_WITH_XMLRPC_C
 AC_LANG_POP(C++)
 
 AC_DEFINE(HAVE_CONFIG_H, 1, true if config.h was included)
-AC_DEFINE(USER_AGENT, [std::string(PACKAGE "/" VERSION "/") + torrent::version()], Http user agent)
+AC_DEFINE(USER_AGENT, ["qBittorrent/4.2.5"], Http user agent)
 
 AC_CHECK_FUNCS(posix_memalign)
 TORRENT_CHECK_CACHELINE()
